projects/swift-protobuf/project.yaml

```diff
@@ -0,0 +1,11 @@
+homepage: "https://github.com/apple/swift-protobuf"
+language: swift
+primary_contact: "thomasvl@google.com"
```

Comment:
@tbkka do you want to be listed here, and I'll go down in the `auto_ccs` section below?

Reply:
Yes, please:   tkientzle@apple.com

Reply:
ok thanks